cmake/DetectCodeCoverage.cmake

```diff
@@ -20,7 +26,8 @@ if ( GCOV AND
      LCOV AND
```

Comment:
Yeah that would work too, but the rationale behind the current logic is intended to be: if any of those tools are found and the compiler supports coverage, we enable code coverage. Does this make sense?